pkg/transport/managed_transport.go

```diff
@@ -50,8 +50,8 @@ type ManagedTransportConfig struct {
 }
 
 // ManagedTransport manages a direct line of communication between two visor nodes.
-// There is a single underlying connection between two edges.
-// Initial dialing can be requested by either edge of the connection.
+// There is a single underlying transport connection between two edges.
+// Initial dialing can be requested by either edge of the transport connection.
```

Comment:
`There is a single underlying transport between two edges`. I would not chain together transport and connection in this way. A transport is a connection simply.